src/rust/src/backend/cipher_registry.rs

```diff
@@ -298,7 +299,9 @@ fn get_cipher_registry(
                 m.add(&idea, &cfb, Some(128), Cipher::idea_cfb64())?;
             }
 
-            m.add(&arc4, none_type.as_any(), None, Cipher::rc4())?;
+            #[cfg(not(CRYPTOGRAPHY_OSSLCONF = "OPENSSL_NO_RC4"))]
+                m.add(&arc4, none_type.as_any(), None, Cipher::rc4())?;
+            }
```

Comment:
You've got a syntax error here, you want:
```suggestion
            #[cfg(not(CRYPTOGRAPHY_OSSLCONF = "OPENSSL_NO_RC4"))]
            m.add(&arc4, none_type.as_any(), None, Cipher::rc4())?;
```